deployment/arm/region.json

```diff
@@ -440,58 +424,6 @@
                 ]
             }
         },
-        {
-            "type": "Microsoft.ApiManagement/service/diagnostics",
```

Comment:
Why are we deleting the APIM child resources here?

Reply:
Same question - How do we ensure the APIM instances are still hoooked up to AI ?

Reply:
Figuring it out. I tried to move these bits to global.json and used copy() function to create these resourced but it wont work as the deployment is per region.